blaze-bs4/toast/toast.js

```diff
@@ -0,0 +1,104 @@
+import { Template } from 'meteor/templating'
+import { Random } from 'meteor/random';
+import { BlazeBs4 } from 'meteor/jkuester:blaze-bs4';
+
+import './toast.html';
+
+BlazeBs4.toast.add = function({ label = "test", body = "Toast Body", autohide = true, delay = 5000, headerBg = "warning", headerFg = "light", transparent = true, small = '', showId = false, width = 'inherit', animation = true, prepend = true }) {
+  const self = this;
+  var id = Random.id();
+  if (showId) small = id;
+  const toast = {
+    id: id,
+    active: true,
+    rendered: false,
+    label: label,
+    bg: headerBg,
+    fg: headerFg,
+    autohide: autohide,
+    delay: delay,
+    body: body,
+    small: small,
+    transparent: transparent,
+    width: width,
+    animation: animation,
+  };
+  if (self.parentNode) {
+    const nextNode = document.getElementsByClassName('toast');
+    var view = null;
+    if(nextNode.length && prepend === true){
+      view = Blaze.renderWithData(Template.toast_entry, toast, self.parentNode, nextNode[0]);
+    }else{
+      view = Blaze.renderWithData(Template.toast_entry, toast, self.parentNode);
+    }
+    if (BlazeBs4.toast.debug) console.log(`created toast ${toast.id}`, view);
+  } else {
+    console.error(`BlazeBs4.toast: no parentNode`, self);
+  }
+};
+
```

Comment:
Looks good from the logic part. However, please format the code using our integrated linter. The examples project contains a `standard` dev dependency, which you can also use to lint the package.

If your run `meteor npm run lint` in the examples if will run the linter for the examples project.

You can also add a new script `"lint-package": "cd ../blaze-bs4 && lint"` if you don't want to install standard globally.

Reply:
looks good to me